test/functional/apps/visualize/_tsvb_chart.ts

```diff
@@ -117,7 +117,7 @@ export default function({ getService, getPageObjects }: FtrProviderContext) {
       });
 
       // FLAKY: https://github.com/elastic/kibana/issues/43150
```

Comment:
Can you just remove this FLAKY comment also?

Reply:
for sure, thx!